app/src/androidTest/java/org/mozilla/focus/activity/AddtoHSTest.java

```diff
@@ -53,14 +55,16 @@ protected void beforeActivityLaunched() {
                     .apply();
 
             webServer = new MockWebServer();
+            webServerPort = webServer.getPort();
+            webServerBookmarkName = "localhost_" + Integer.toString(webServerPort);
 
             try {
                 webServer.enqueue(new MockResponse()
                         .setBody(TestHelper.readTestAsset("plain_test.html")));
                 webServer.enqueue(new MockResponse()
                         .setBody(TestHelper.readTestAsset("plain_test.html")));
 
-                webServer.start();
+                //webServer.start();
```

Comment:
@rpappalax so is webserver starting somewhere else, since this is commented out?